tasking/pulp/tasking/constants.py

```diff
@@ -4,6 +4,20 @@
     # The interval in seconds for which the Celery Process monitor thread sleeps between
     # checking for missing Celery processes.
     CELERY_CHECK_INTERVAL=60,
+    # The interval in seconds during which the Celery Processes being monitored need to report a
+    # hearbeat to be considered active.
+    CELERY_TIMEOUT_SECONDS=200,
```

Comment:
Was this set to 200 before? I remember the age-out time being 300 seconds. Generally I want to lower these timing for Pulp3 anyway. Should we do that now or in another PR? I'm OK either way.